aprotos/07_CSS2/hw_5flex.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+   <meta charset="utf-8">
+   <title>Flex1</title>
+   <style>
+      .containter {
+         display: flex;
+         align-items: flex-start;
+      }
+      .navigation {
+         display: flex;
+         flex-direction: column;
+         flex: 0 0 15em;
+         background-color: purple;
+      }
+      .content {
+         margin-bottom: .5em;
+         background-color: green;
+         display: flex;
+         flex-direction: column;
+         flex: 0 0 80%;
+         
+      }
+      .lowwer {
+         background-color: grey;
+         display: flex;
+         flex: 1 1 100%;
+      }
+      .send {
+         display: flex;
```

Comment:
Not needed here (flex), use float

Reply:
why not flex? if i delete and use float my pictrure make size = all width

Reply:
for picture need just float